internal/version/version.go

```diff
@@ -9,7 +9,7 @@ import "runtime/debug"
 
 const (
 	// tag specifies the current release tag. It needs to be manually updated.
-	tag       = "v1.0.3"
+	tag       = "v1.0.4"
```

Comment:
Was not sure what the correct value should be. Is this a patch release or minor release for orchestrion? Is the answer determined by whether we're upgrading minor or patch versions of dd-trace-go (in this case, the latter)?

Please clarify and I can update instructions in the template: https://datadoghq.atlassian.net/wiki/spaces/DL/pages/2477949158/Template+v+MAJOR+.+MINOR+.+PATCH+Release+Checklist#Validate-the-release

Reply:
Usually we do a patch release with these.